Svc/PrmDb/test/ut/PrmDbImplTester.cpp

```diff
@@ -588,290 +519,100 @@ void PrmDbImplTester::runFileReadError() {
         ASSERT_CMD_RESPONSE_SIZE(1);
         ASSERT_CMD_RESPONSE(0,PrmDbImpl::OPCODE_PRM_SAVE_FILE,12,Fw::CmdResponse::EXECUTION_ERROR);
 
-        Os::clearOpenInterceptor();
```

Comment:
Where has the function of all this deleted code gone?

Reply:
The open interceptor wasn't doing anything special.  Just returning known specific statuses.  `Os::Stub::File::Test::TestFile` already supports that built-in, so the code was switched to use that feature rather than reimplementing it.